### PR TITLE
README.md expand import examples

### PR DESCRIPTION
**Pull request by mb21**
Proposal to expanded the README to what I was looking for to get started. Feel free to change.

I was also not sure what this sentence means:

> Because the `Iterator.prototype` methods depend on a receiver (the `this` value), the main export in each subdirectory takes the string to operate on as the first argument.

Does it relate to explicit imports instead of shimming or is it an implementation detail?

### CODE REVIEW

**Comment by ljharb** [comment]
It means that instead of forcing you to do, eg, `require('es-iterator-helpers'/Iterator.prototype.drop').call(iterator, count)`, you'd do `require('es-iterator-helpers'/Iterator.prototype.drop')(iterator, count)`. "The string" there is a mistake; it should be "the iterator".

**Review by ljharb — APPROVED** [review]
This is fine, but if you're using node you need to be able to look at either CJS or ESM examples and implicitly understand how to convert one to the other.

**Comment by mb21** [comment]
Right, I mean I figured it out eventually what the require path was relative to, how to convert CJS to ESM and which module executed the shims automagically. But all three together were a bit many hurdles for my taste, hence this PR.

I pushed two commits (to make CI pass and to fix the typo you mentioned), so now it's ready to merge I think.

**Comment by ljharb** [comment]
Removing evalmd is like deleting tests - the benefit of having an ESM example is much much smaller than the benefit of guaranteeing readme examples are correct.

That said, i think evalmd has an html comment way to ignore specific code blocks, so that might work here.

**Comment by mb21** [comment]
Done...

File: README.md [hunk 1]
```diff
@@ -67,10 +69,22 @@ assert.deepEqual(
 );
 ```
 
+Shim using `require`:
+
+[](#preventEval)
```
Comment (ljharb) [inline README.md:74]:
```suggestion
```
these should still be evaluable, no?

**Review by ljharb — APPROVED** [review]

**Pull request merged**